### PR TITLE
NEW: Start on a new dtypes section

### PR DESCRIPTION
**Pull request by seberg**

### CODE REVIEW

File: references.bib [hunk 1]
```diff
@@ -197,6 +197,51 @@ @inproceedings{harrington2009scipy
...
+  volume = {3},
+  number = {28},
+  pages = {809},
+  author = {Nathan J. Goldbaum and John A. ZuHone and Matthew J. Turk and Kacper Kowalik and Anna L. Rosen},
+  title = {unyt: Handle,  manipulate,  and convert data with units in Python},
+  journal = {Journal of Open Source Software}
+}
+
+@online{pint,
+  title = {Pint: Operate and manipulate physical quantities in Python}
```
Comment (jarrodmillman) [inline references.bib]: Missing `,`s at the end of the title and author fields.

File: references.bib [hunk 2]
```diff
@@ -197,6 +197,51 @@ @inproceedings{harrington2009scipy
...
+  author = {{Pint Developers}}
+  year = {2012},  % First release?
```
Comment (jarrodmillman) [inline references.bib]: `%` isn't a comment character in bib files.

File: references.bib [hunk 3]
```diff
@@ -197,6 +197,51 @@ @inproceedings{harrington2009scipy
...
+  url = {https://github.com/hgrecco/pint},
+  urldate = {2019-12-17}
+  }
+
+@online{pyadolc,
+  title = {PYADOLC: Python Bindings to ADOL-C}
```
Comment (jarrodmillman) [inline references.bib]: Missing ,s at the end of the title and author fields.

Comment (seberg) [reply references.bib]: opst, should have compiled with references...

**Comment by jarrodmillman** [comment]
Unless you object, I will merge this later today. Once we have a complete rough draft and know where we are submitting this, we can revisit this.  You can also make a new PR whenever you hear back from pygeos.

**Comment by seberg** [comment]
Of course, go ahead and merge whenever you like. I understand the idea was to get something rough in there quickly. If I should rework a bit more, happy to do that before or after though.

**Pull request merged**